packages/calcite-components/src/components/panel/panel.stories.ts

```diff
@@ -440,3 +440,24 @@ export const withNoHeaderBorderBlockEnd_TestOnly = (): string =>
   html`<calcite-panel style="--calcite-panel-header-border-block-end:none;" height-scale="s" heading="My Panel"
     >Slotted content!</calcite-panel
   >`;
+
+export const contentTopSlot_TestOnly = (): string => html`
+  <div style="height: 300px; width: 400px; display: flex">
+    <calcite-panel height-scale="s">
+      <calcite-action-bar slot="action-bar">
+        <calcite-action-group>
+          <calcite-action text="Add" icon="plus"> </calcite-action>
+          <calcite-action text="Save" icon="save"> </calcite-action>
+          <calcite-action text="Layers" icon="layers"> </calcite-action>
+        </calcite-action-group>
+      </calcite-action-bar>
+      <div slot="content-top" id="content-top">Slot for a content-top.</div>
+      <div slot="header-content">Header!</div>
+      <p>Slotted content!</p>
+      <p style="height: 400px">Hello first!</p>
```

Comment:
Is this the simplest test case or can we simplify by removing these paragraphs with inline styles?

Reply:
I can remove inline styles, but paragraphs are there to show the sticky property of `content-top` when scrolled.